sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -52,195 +53,6 @@
 _LOGGER = logging.getLogger(__name__)
 
 
```

Comment:
I must apologize for this massive diff block.  I thought it'd be smart to move this chuck downstream a bit to keep session stuff near each other and core receiver bits up top (contemplating even moving it to its own file) but obviously diff isn't too happy with that.  Nothing changed within ServiceBusSession other than location, feel free to skim/ignore that.